src/DocField.js

```diff
@@ -4,13 +4,15 @@ class DocField extends Component {
   render() {
     const prefix = this.props.repoName + " / doc / goals /"
     const filepath =  (
+      <section className="docfield">
       <div>
```

Comment:
We should likely indent the `<div>` inside the `<section>`. What do you think?

Reply:
Done!